test-frame-metrics-collector/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.skodjob</groupId>
+        <artifactId>test-frame</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>test-frame-metrics-collector</artifactId>
+
+    <licenses>
+        <license>
+            <name>The Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <name>${project.groupId}:${project.artifactId}</name>
+    <description>Kubernetes test library</description>
+    <url>https://github.com/skodjob/test-frame</url>
+
+    <scm>
+        <connection>scm:git:git:/github.com/skodjob/test-frame.git</connection>
+        <developerConnection>scm:git:ssh://github.com/skodjob/test-frame.git</developerConnection>
+        <url>https://github.com/skodjob/test-frame</url>
+    </scm>
+
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://github.com/skodjob/test-frame/issues</url>
+    </issueManagement>
+
+    <distributionManagement>
+        <repository>
+            <id>github</id>
+            <name>GitHub Apache Maven Packages</name>
+            <url>https://maven.pkg.github.com/skodjob/test-frame</url>
+        </repository>
+    </distributionManagement>
+
+    <developers>
+        <developer>
+            <id>im-konge</id>
+            <name>Lukáš Král</name>
+            <email>lukywill16@gmail.com</email>
+        </developer>
+        <developer>
+            <id>kornys</id>
+            <name>David Kornel</name>
+            <email>kornys@outlook.com</email>
+        </developer>
+        <developer>
+            <id>Frawless</id>
+            <name>Jakub Stejskal</name>
+            <email>xstejs24@gmail.com</email>
+        </developer>
+        <developer>
+            <id>see-quick</id>
+            <name>Maros Orsak</name>
+            <email>maros.orsak159@gmail.com</email>
+        </developer>
+    </developers>
+
+    <properties>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <!--  CODE dependencies  -->
+        <log4j.version>2.17.2</log4j.version>
+
+        <!--  TEST dependencies  -->
+        <mockito.version>5.11.0</mockito.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.skodjob</groupId>
+            <artifactId>test-frame-common</artifactId>
+        </dependency>
+
+        <dependency>
```

Comment:
we use logback instead of full log4j so I would say you should also use same approach.

Reply:
nevermind I will move to log4j whole modules :)

Reply:
> we use logback instead of full log4j so I would say you should also use same approach.

I tried that but in Strimzi we have banned these dependencies
```java
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.BannedDependencies failed with message:
Found Banned Dependency: ch.qos.logback:logback-classic:jar:1.4.12
Found Banned Dependency: ch.qos.logback:logback-core:jar:1.4.12
Use 'mvn dependency:tree' to locate the source of the banned dependencies.
``` 
So only log4j for me was an option :/